pkg/controller/v1beta1/inferenceservice/reconcilers/deployment/deployment_reconciler.go

```diff
@@ -330,11 +330,11 @@ func addGPUResourceToDeployment(deployment *appsv1.Deployment, targetContainerNa
 	// If CustomGPUResourceTypeAnnotationKey is set, the specified custom GPU resource will be added to the available GPUResourceTypeList.
 	customGPUResourceTypes := deployment.GetAnnotations()[constants.CustomGPUResourceTypesAnnotationKey]
 	if customGPUResourceTypes != "" {
-		constants.GPUResourceTypeList = append(constants.GPUResourceTypeList, strings.Split(customGPUResourceTypes, ",")...)
+		constants.DefaultGPUResourceTypeList = append(constants.DefaultGPUResourceTypeList, strings.Split(customGPUResourceTypes, ",")...)
```

Comment:
This is already done in https://github.com/kserve/kserve/pull/4303/files#diff-81ddbadfb415ccbb9c7af84f11668d1aa5e53c34025bf86d4702f16b4e42f045R364 right?